### PR TITLE
Add Open Service Broker as an example

### PR DESCRIPTION
**Pull request by meip**

### CODE REVIEW

File: gettingStarted.md [hunk 1]
```diff
@@ -42,8 +42,15 @@ sample implementation has been provided for
 
 [MySQL Java Broker](https://github.com/cloudfoundry-community/cf-mysql-java-broker):
 A Java port of the Ruby-based
+
 [MySQL broker](https://github.com/cloudfoundry/cf-mysql-broker) above.
 
+[Open Service Broker](https://github.com/swisscom/open-service-broker):
```
Comment (duglin) [inline gettingStarted.md]: Could we add "Swisscom" to the name since "Open Service Broker" is a pretty generic name?

Comment (avade) [reply gettingStarted.md]: +1 to this suggestion. I have heard confusion about the project name from others in the community.

**Comment by duglin** [comment]
looks good,just need a CLA

**Comment by meip** [comment]
@duglin Swisscom signed the CLA, please verify over my Github organizations.

**Comment by duglin** [comment]
ah, I just had to kick it - thanks!

**Comment by duglin** [comment]
LGTM

**Comment by pmorie** [comment]
LGTM

**Comment by vaikas-google** [comment]
LGTM

**Comment by avade** [comment]
lgtm

**Comment by pmorie** [comment]
I do think it would be nice to disambiguate whether the linked project is an SDK, generic broker, or something else.

**Pull request merged**